optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/test/java/org/optaplanner/quarkus/OptaPlannerProcessorSolverPropertiesTest.java

```diff
@@ -42,13 +43,13 @@
 
     @RegisterExtension
     static final QuarkusUnitTest config = new QuarkusUnitTest()
-            .overrideConfigKey("quarkus.optaplanner.solver.environment-mode",
-                    "FULL_ASSERT")
+            .overrideConfigKey("quarkus.optaplanner.solver.daemon", "true")
+            .overrideConfigKey("quarkus.optaplanner.solver.domain-access-type", "REFLECTION")
+            .overrideConfigKey("quarkus.optaplanner.solver.environment-mode", "FULL_ASSERT")
             .overrideConfigKey("quarkus.optaplanner.solver.move-thread-count", "2")
             .overrideConfigKey("quarkus.optaplanner.solver.termination.spent-limit", "4h")
             .overrideConfigKey("quarkus.optaplanner.solver.termination.unimproved-spent-limit", "5h")
             .overrideConfigKey("quarkus.optaplanner.solver.termination.best-score-limit", "0")
-            .overrideConfigKey("quarkus.optaplanner.solver.domain-access-type", "REFLECTION")
```

Comment:
Nitpick:
in the SolverConfig.java, domain-access-type is before termination, but after move-thread-count. Can we keep that order here too?